checks/51-rpmlint

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+RPMLINT_CHROOT="$BUILD_ROOT/usr/lib/rpmlint-image"
+WORKDIR=$RPMLINT_CHROOT/RPM
+SOURCES="$BUILD_ROOT/home/abuild/rpmbuild/SOURCES"
+
+find $BUILD_ROOT/home/abuild/rpmbuild/RPMS -type f -name '*.rpm' -exec cp {} $WORKDIR \;
+find $BUILD_ROOT/home/abuild/rpmbuild/SRPMS -type f -name '*.rpm' -exec cp {} $WORKDIR \;
+
+find $SOURCES -name "*rpmlintrc" -exec cp {} $WORKDIR \;
+RPMLINTRC_OPTION=$(ls $WORKDIR | grep rpmlintrc | sed "s,$WORKDIR,,")
```

Comment:
The code in OBS doesn't seem to handle rpmlintrc at all, so rpmlint seems to do that itself?

Reply:
Rpmlint 1.x has some auto-loading mechanism. So we should rather pass it as an argument for rpmlint2.